src/ranch.erl

```diff
@@ -193,24 +193,33 @@ start_error(_, Error) -> Error.
 
 -spec stop_listener(ref()) -> ok | {error, not_found}.
 stop_listener(Ref) ->
+	erpc:call(node(), fun() -> stop_listener1(Ref) end, 5000).
```

Comment:
I would put a larger timeout. I know the supervisor calls are `infinity`, but that we likely don't have too much traffic on this supervisor, so it should go fast. But heavily loaded systems may make this reach the timeout. Maybe 30s?

Also the timeout error must be handled and we return `{error, timeout}` instead of an exception. And we'll also want to explain in the documentation that this can happen and why.

Reply:
Already changed it, and also added an explanation. I picked the maximum possible timeout, `16#ffffffff`, since if it wasn't for that peculiar `erpc:call` behavior, we would have gone with the default `infinity`.

Reply:
Let's put it at 5 minutes and not handle the timeout after that. If something is wrong we want to be informed, not wait 50 days before we have a weird crash in the logs.

Reply:
Done.